### ISSUE TITLE
Suggestion: include information on how to clear the interval on demand.

### ISSUE THREAD

**Issue by jay-jlm** [react-corekit/use-interval#1]
A common scenario is to have the need to clear the interval without unmounting the whole component.
I'm currently using the following code do clear the interval on demand, it would be nice to have some official information on how to do it:

```
const [isRunning, setIsRunning] = useState(true);

  useInterval(() => {
    setCount(count + 1);
  }, isRunning ? delay : null);
```

**Comment by glongh** [comment]
Hi @jay-jlm thank you for your suggestion, I will update the documentation.

**Comment by glongh** [comment]
Released as 1.1.1, Thanks again @jay-jlm .
